client/app/templates/invoice.hbs

```diff
@@ -27,15 +27,14 @@
     </tr>
   </thead>
   <tbody>
-    {{#each @model.dcpInvoiceApplications as |dcpApplication|}}
-    <tr>
-      <td>{{dcpApplication.dcpApplicationNumber}}</td>
-      <td>{{dcpApplication.dcpAction}}</td>
-      <td class="text-right font-mono">{{dcpApplication.dcpFee}}</td>
-    </tr>
+    {{#each @model.lineitems as |lineitem|}}
+      <tr>
+        <td>{{lineitem.dcp-applicationnumber}}</td>
+        <td>{{lineitem.dcp-action}}</td>
+        <td class="text-right font-mono">{{lineitem.dcp-fee}}</td>
```

Comment:
this works?

Reply:
@godfreyyeung yeah why not? it doesn't camelize bc it's not a real model. the json api serializer does convert it, however, because it does so recursively for all object attr names.